examples/ssd/example_outputs.md

```diff
@@ -0,0 +1,139 @@
+# SSD Example Outputs
+
+### Dog Beach
+
+![dog beach](https://farm9.staticflickr.com/8184/8081332083_2a9a971149_o_d.jpg)
+```bash
+curl -o dogbeach.jpg https://farm9.staticflickr.com/8184/8081332083_2a9a971149_o_d.jpg
+curl -X POST http://127.0.0.1:8080/ssd/predict -F "data=@dogbeach.jpg"
+{
+  "prediction": [
+    [
+      "dog",
+      2484,
+      1300,
+      2996,
+      1741
+    ],
+    [
+      "person",
+      1513,
+      1580,
+      1846,
+      2584
+    ],
+    [
+      "person",
+      810,
+      1561,
+      1076,
+      2161
+    ],
+    [
+      "person",
+      1151,
+      86,
+      1403,
+      732
+    ],
+    [
+      "person",
+      3934,
+      228,
+      4169,
+      766
+    ],
+    [
+      "dog",
+      591,
+      416,
+      976,
+      722
+    ],
+    [
+      "horse",
```

Comment:
does it matter that it picks up the wrong thing, or is that good for discussion?